maven-plugins/pom.xml

```diff
@@ -9,8 +9,8 @@
         <relativePath>../parent/pom.xml</relativePath>
     </parent>
     <groupId>com.yahoo.vespa</groupId>
-    <artifactId>bundle-plugins</artifactId>
-    <name>bundle-plugins</name>
+    <artifactId>maven-plugins</artifactId>
+    <name>maven-plugins</name>
```

Comment:
I don't think `name` is needed, so I suggest to remove it.

Reply:
Great, I was hoping you would say that.

Reply:
Fixed

Reply:
👍